rsync_to_rdisc.py

```diff
@@ -22,35 +22,30 @@ def find_owner(filename):
     return getpwuid(os.lstat(filename).st_uid).pw_name
 
 def make_mail(filename, state):
-    if state[0] == "ok": 
-        if find_owner(filename) in settings.owner_dic:
-            email_to = [settings.owner_dic[find_owner(filename)]]
-        else:
-            email_to = settings.finished_mail
-
+    file_owner = find_owner(filename)
+    if file_owner not in settings.owner_dic:
+        email_to = [settings.finished_mail]
+        subject = 'USER WARNING: user {} does not exist'.format(file_owner)
+        text = "<html><body><p>" + 'User {} does not exist in settings.py. Please add.'.format(file_owner) + "</p></body></html>"
```

Comment:
I think this should be separate from the 'state' check emails. And email state emails to settings.finished_mail if file_owner is not found?